package/chalicelib/reportGenerator.py

```diff
@@ -328,18 +413,39 @@ def send_individual_report(self, user, recipients=None):
             processed = self.process_api_response(response)
             response_by_account[acct_num] = processed
 
+        total = ReportGenerator.sum_dictionary(response_by_account)
+        response_by_account["Total"] = total
+
         # Create graphics.
         for acct in response_by_account:
-            if response_by_account[acct][user]:
-                plt = GraphGenerator.graph_individual(user, response_by_account[acct][user])  # create graphical reports
-                plt[0].savefig("images/%s/%s.png" % (user, acct), bbox_extra_artists=(plt[1],), bbox_inches='tight', dpi=200)
+            if user in response_by_account[acct]:  # create graphical reports
```

Comment:
Another function for this too. Just for consistency with using a function in ReportGenerator.send_management_report()

Reply:
done